data/pagination.go

```diff
@@ -70,49 +80,55 @@ func getLimitFromURLQuery(ctx context.Context, cfg *config.Config, query url.Val
 
 func getPageFromURLQuery(ctx context.Context, cfg *config.Config, query url.Values) int {
 	pageParam := query.Get("page")
-
 	if pageParam == "" {
 		return cfg.DefaultPage
 	}
 
 	page, err := strconv.Atoi(pageParam)
 	if err != nil {
 		log.Info(ctx, "unable to convert search page to int - set to default", log.Data{
-			"default": cfg.DefaultPage,
-			"page":    page,
+			"default":   cfg.DefaultPage,
+			"pageParam": pageParam,
 		})
 		query.Set("page", strconv.Itoa(cfg.DefaultPage))
-		page = cfg.DefaultPage
+		return cfg.DefaultPage
 	}
 
-	if page < 1 {
-		log.Info(ctx, "page number is less than default - set to default", log.Data{
+	if page < cfg.DefaultPage {
+		log.Info(ctx, "page number is less than 1 - set to default", log.Data{
 			"default": cfg.DefaultPage,
 			"page":    page,
 		})
 		query.Set("page", strconv.Itoa(cfg.DefaultPage))
-		page = cfg.DefaultPage
+		return cfg.DefaultPage
```

Comment:
These changes seem contradictory. 

`if page < cfg.DefaultPage`

then log message should be

`page number less than default` - otherwise what happens when we change the `defaultPage` to not 1. 

Should be consistency with the log message above as well with `page` vs `pageParam`